app/src/pages/inside/testCaseLibraryPage/testCaseFolders/modals/duplicateFolderModal/duplicateFolderModal.tsx

```diff
@@ -96,12 +95,7 @@ const DuplicateFolderModal = reduxForm<DuplicateFolderFormValues, DuplicateFolde
       ? null
       : coerceToNumericId(values.destinationFolder?.id);
 
-    dispatch(
-      duplicateFolderAction({
-        folderName: values.folderName,
-        ...(parentFolderId ? { parentFolderId } : {}),
-      }),
-    );
+    duplicateFolder(folderId, values.folderName, parentFolderId).catch(noop);
```

Comment:
```suggestion
    duplicateFolder(folderId, values.folderName, parentFolderId);
```

Seems like it can be removed as all possible errors are handled in `duplicateFolder`